### PR TITLE
Added 15+ new crates, fixed issues raised in #9901

### PR DESCRIPTION
**Pull request by Owai-Seek**
Remake of previous PR that I broke due to inexperience.  #9901 Also apparently #9905 needs this to work.
Added 15+ new crates to cargo.
Re-ordered some some of the crates.

Added the following:

/datum/supply_pack/misc/randombedsheets
/datum/supply_pack/misc/coloredsheets
/datum/supply_pack/misc/cbtpack
/datum/supply_pack/misc/exoticfootwear
/datum/supply_pack/misc/shower
/datum/supply_pack/misc/casinocrate
/datum/supply_pack/misc/coincrate
/datum/supply_pack/organic/fiestatortilla
/datum/supply_pack/organic/fakemeat
/datum/supply_pack/organic/mixedboxes
/datum/supply_pack/organic/monkeydripmeat
/datum/supply_pack/emergency/medemergencylite
/datum/supply_pack/emergency/medemergency
/datum/supply_pack/service/buildabar
/datum/supply_pack/service/maintgarden
/datum/supply_pack/service/janitor/advanced
/datum/supply_pack/critter/mouse
/datum/supply_pack/medical/bodybags

and probably a few more I may have forgotten about.

## About The Pull Request

In my attempt to clean up the categories in cargo I instead make the problem worse by adding even more crates to buy. (I'll probably be adding even more and moving stuff around at a later date, but I had to get these out before I burnt out on it.)

## Why It's Good For The Game

More Crates

## Changelog
:cl:
add: 15+ new crates for cargo
tweak: organizes crates and moving them to proper categories
spellcheck: fixed a few typos
del: some dumb stuff like toner crates
re
/:cl:

### CODE REVIEW

File: code/modules/cargo/packs/emergency.dm [hunk 1]
```diff
@@ -44,8 +44,28 @@
 	crate_name = "emergency crate"
 	crate_type = /obj/structure/closet/crate/internals
 
+/datum/supply_pack/emergency/medemergency
+	name = "Emergency Medical Supplies"
+	desc = "Emergency supplies for a front-line medic. Contains two boxes of body bags, a medical HUD, a defib unit, medical belt, toxin bottles, epipens, and several types of medical kits."
+	cost = 3000
```
Comment (Trilbyspaceclone) [inline code/modules/cargo/packs/emergency.dm]: oooh this seems like a good "DropPod Only"

Comment (Owai-Seek) [reply code/modules/cargo/packs/emergency.dm]: I went ahead with your suggestion but during testing I found out that drop pods cant be used with defib units. (at least the loaded variant). I was sick of testing at that point, so I boosted the price to 6600 and made it available for regular purchase.

File: code/modules/cargo/packs/misc.dm [hunk 2]
```diff
@@ -143,6 +115,136 @@
...
+/datum/supply_pack/misc/coincrate
+	name = "Coin Crate"
+	desc = "Psssst, hey, you. Yes, you. I've heard that coins can do some special things on your station, give you access to some pretty cool stuff. Here's the deal, you give me some credits, and I give so some coins. Sound like a deal? I'll give you 10 for 10000 creds."
+	contraband = TRUE
+	cost = 10000
+	contains = list(/obj/item/coin/iron)
+	crate_name = "coin crate"
+	crate_type = /obj/structure/closet/crate/large
+
+/datum/supply_pack/misc/coincrate/generate()
```
Comment (Trilbyspaceclone) [inline code/modules/cargo/packs/misc.dm]: Make them random and lower the cost to 4k~

File: code/modules/cargo/packs/service.dm [hunk 3]
```diff
@@ -83,8 +70,103 @@
...
+					/obj/item/storage/box/lights/mixed,
+					/obj/item/lightreplacer)
+	crate_name = "replacement lights"
+
+/datum/supply_pack/service/janitor/advanced
+	name = "Advanced Sanitation Crate"
+	desc = "Contains all the essentials for an advanced spacefaring cleanup crew. This kit includes a trashbag of holding, an advanced mop, a bottle of space cleaner, a floor buffer, and a holosign projector."
+	cost = 5000
+	access = ACCESS_JANITOR
+	contains = list(/obj/item/storage/bag/trash/bluespace,
```
Comment (Trilbyspaceclone) [inline code/modules/cargo/packs/service.dm:161]: Hmm...

File: code/modules/cargo/packs/misc.dm (hunk 2 above)
Comment (Owai-Seek) [reply code/modules/cargo/packs/misc.dm]: There are coins that can go up to 3000 credits. The crate isn't supposed to be particularly practical or worthwhile it's meant to get coins explicitly for use in a vendor. Making higher value coins possible in the crate just makes it easier for people to get -even more- out of it by shoving a high value coin in a slot machine, getting a refund, and getting dozens upon dozens of extra coins.

Honestly I'm not sure if the slot machine dumps out iron or silver coins, but either way, vendors only take one coin of any value for their extra content. (AFAIK).

File: code/modules/cargo/packs/service.dm (hunk 3 above)
Comment (Owai-Seek) [reply code/modules/cargo/packs/service.dm:161]: Is it the access, cost, or the trashbag? I can unlist it from janitor and make it a bit more expensive, or replace the bag with a normal one.

In all honesty, most of the time when I play janitor I don't even need a bluespace bag unless someone is throwing glowsticks everywhere.

**Review by Putnam3145 — APPROVED** [review]

File: code/modules/cargo/packs/engineering.dm [hunk 4]
```diff
@@ -121,6 +121,17 @@
...
+	name = "Power Tools Crate"
+	desc = "Setting up the aux base? Working out in space? Maybe you just hate doing things by hand. Grab some power tools and slash your workload in half! Contains a hardhat, hazard vest, Hand Drill, and Jaws of Life. Requires Engineering Access to open."
+	cost = 6500
+	access = ACCESS_ENGINE
+	contains = list(/obj/item/clothing/head/hardhat,
+					/obj/item/clothing/suit/hazardvest,
+					/obj/item/screwdriver/power,
+					/obj/item/crowbar/power)
+	crate_name = "power tools crate"
+	crate_type = /obj/structure/closet/crate/secure/engineering
```
Comment (ghost) [inline code/modules/cargo/packs/engineering.dm]: sick

File: code/modules/cargo/packs/misc.dm (hunk 2 above)
Comment (ghost) [reply code/modules/cargo/packs/misc.dm]: you can already mass produce coins, 5 coins per 1 sheet of material

**Review by silicons — APPROVED** [review]

**Pull request merged**